server/accounts_test.go

```diff
@@ -1003,6 +1004,30 @@ func TestAccountMapsUsers(t *testing.T) {
 	}
 }
 
+func TestAccountGlobalDefault(t *testing.T) {
+	opts := defaultServerOptions
+	s := New(&opts)
+
+	acc := s.LookupAccount(globalAccountName)
```

Comment:
This and 2 other below could be one-liners.

Reply:
will fix.